### DISCUSSION TITLE
Soft Delete by Default (or config)

### DISCUSSION THREAD

**Discussion by iRaziul** [awcodes/filament-curator#436] [Ideas]
I am using this plugin a lot and I find it useful to have soft-deletes integrated into it.
We can have it by default or use config to enable/disable soft delete.

**Comment by iRaziul** [comment]
@awcodes What do you think about this? 🤔

**Comment by awcodes** [comment]
Let me think about it some.

This could get tricky since it also involves deleting files and directories from the system and not just records in the database.

**Reply by iRaziul** [reply]
Maybe we can move the files to another directory or make them private!!

**Reply by awcodes** [reply]
Possibly, just don't want to make a quick decision on this. Want to think it through completely.

**Reply by iRaziul** [reply]
Sure 😀
